### PR TITLE
Pdf2json remove

### PR DESCRIPTION
**Pull request by marianorodriguez**
Removed pdf2json module and all its references, as we decided to have pdfminer as the (for now) only pdf extractor.

- **Some pdf2json references can still be found on the Dockerfile**. As this is coming from AxaTechLab repository, I'm not sure about what to do with that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -339,8 +327,6 @@ The following _required_ dependencies need to be installed for Parsr to work pro
 
 Depending upon the type of documents to be treated by the platform, one or multiple of the following dependencies should be installed.
 
-If simple PDFs containing digital (or _selectable_) textual elements are to be fed into the system, the **`pdf2json`** library needs to be installed.
```
Comment (aarohijohal) [inline README.md]: I would change that to:
> If simple PDFs containing digital (or _selectable_) textual elements are to be fed into the system, the **`pdfminer`** library needs to be installed.

File: README_zh-cn.md [hunk 2]
```diff
@@ -314,8 +301,6 @@ Parsr 默认的 OCR 解决方案是 tesseract，这是 Parsr 的基本依赖。
 
 根据平台要处理的文档类型，应安装以下一个或多个依赖项。
 
-如果要将包含数字（或*可选定*）文本元素的简单 PDF 输入系统，则需要安装**`pdf2json` **库。
```
Comment (aarohijohal) [inline README_zh-cn.md]: 如果要将包含数字（或*可选定*）文本元素的简单 PDF 输入系统，则需要安装**`pdfminer` **库。

File: README.md [hunk 3]
```diff
@@ -117,18 +116,7 @@ pip install ghostscript camelot-py
 4. For table detection, install [**camelot**](https://camelot-py.readthedocs.io/en/master/user/install-deps.html#for-windows).
 5. For the **pdfminer** extractor for pdfs, [follow these steps](https://github.com/pdfminer/pdfminer.six#how-to-install).
```
Comment (aarohijohal) [inline README.md]: This line should be at number 2 or 3 in the list.

Comment (marianorodriguez) [reply README.md]: I just figured that in README_zh-cn this list is incomplete

**Comment by aarohijohal** [comment]
Thanks @marianorodriguez !

**Pull request merged**